src/ui/dialogs/multiUserMemberlist.ts

```diff
@@ -0,0 +1,98 @@
+import Dialog from '../Dialog'
+import MultiUserContact from '../../MultiUserContact'
+import Translation from '../../util/Translation'
+
+
+let multiUserMemberlist = require('../../../template/multiUserMemberlist.hbs');
+
+export default function(multiUserContact: MultiUserContact) {
+   let content = multiUserMemberlist({});
+
+   let dialog = new Dialog(content);
+   let dom = dialog.open();
+   let connection = multiUserContact.getAccount().getConnection();
+   let items;
+
+   dom.find('form').on('submit', (ev) => {
+      ev.preventDefault();
+      let saveitems = $('.jsxc-memberlist-textarea').val().toString().split('\n');
+      let i=0;
+      //check old items if there were deleted
+      for (;i<items.length;i++)
+      {
+          let n=0;
+          let found=false;
+          for (;n<saveitems.length;n++)
+          {
+              if (saveitems[n].trim().length>0&&$(items[i]).attr('jid')===saveitems[n].trim())
+              {
+                  found=true;
+                  break;
+              }
+          }
+          if (!found)
```

Comment:
No need to loop through all items. Just use `saveItems.indexOf(items[i]) < 0`.

Reply:
saveitems is a string array and items is a list of xml objects... dont think indexOf will do the job!?

Reply:
`items[i].attr('id') && saveItems.indexOf(items[i].attr('id')) < 0` should be the same as the inner loop.